packages/custom-field/src/vaadin-custom-field.js

```diff
@@ -222,6 +225,9 @@ class CustomField extends FieldMixin(FocusMixin(KeyboardMixin(ThemableMixin(Elem
     this.__observer = new FlattenedNodesObserver(this.$.slot, () => {
       this.__setInputsFromSlot();
     });
+
+    this._tooltipController = new TooltipController(this);
```

Comment:
Would there be a viable way to have the custom field's `shouldShow` return `false` if one of the child components' `shouldShow` returns false?

https://user-images.githubusercontent.com/1222264/188623283-2fdc2de1-ebfb-47a0-af3d-d88deb8c0e24.mp4

Reply:
Good point. Would it be enough to check for `opened` state for now?

Reply:
Added `setShouldShow` and updated the integration test accordingly.